tests/tests_unit/test_api/test_data_modeling/test_instances.py

```diff
@@ -33,3 +39,116 @@ def test_instances_api_dump_instance_source(sources, expected):
     # ViewIdentifier = Union[ViewId, Tuple[str, str], Tuple[str, str, str]]
     # ViewIdentifier | Sequence[ViewIdentifier] | View | Sequence[View]
     assert expected == InstancesAPI._dump_instance_source(sources)
+
+
+@pytest.fixture
+def empty_node_dump():
+    return {
+        "space": "space",
+        "externalId": "xid",
+        "version": 1,
+        "lastUpdatedTime": 2,
+        "createdTime": 3,
+    }
+
+
+@pytest.fixture
+def empty_edge_dump():
+    return {
+        "space": "space",
+        "externalId": "xid",
+        "version": 4,
+        "lastUpdatedTime": 5,
+        "createdTime": 6,
+        "type": {"space": "foo-space", "externalId": "bar-xid"},
+        "startNode": {"space": "the-space", "externalId": "the-xid"},
+        "endNode": {"space": "the-space", "externalId": "the-xid"},
+    }
+
+
+@pytest.mark.parametrize("instance_type", (Node, Edge))
```

Comment:
I would expect these test to be in the `tests/tests_unit/test_data_classes/test_data_models/test_instances.py`